test/Libraries/RevitIntegrationTests/ElementTests.cs

```diff
@@ -155,5 +155,24 @@ public void CanCheckIfTwoElementsAreJoined()
             Assert.AreEqual(true, isJoinedTrue);
             Assert.AreEqual(false, isJoinedFalse);
         }
+
+        [Test]
+        [TestModel(@".\Element\elementJoin.rvt")]
+        public void CanJoinTwoIntersectingElements()
+        {
+            // Arange
+            string samplePath = Path.Combine(workingDirectory, @".\Element\canJoinTwoIntersectingElements.dyn");
+            string testPath = Path.GetFullPath(samplePath);
+            int expectedElementId = 208259;
+
+            ViewModel.OpenCommand.Execute(testPath);
+            RunCurrentModel();
+
+            // Act 
+            var firstJoinedElementId = GetPreviewValue("0ee537c473c04470b5041d16d9b5ab12");
```

Comment:
let's also check that 2 elements that were not intersecting puts the node in error state or returns nothing